Formula/staq.rb

```diff
@@ -0,0 +1,34 @@
+class Staq < Formula
+  desc "Full-stack quantum processing toolkit"
+  homepage "https://github.com/softwareQinc/staq"
+  url "https://github.com/softwareQinc/staq/archive/v3.0.1.tar.gz"
+  sha256 "306e46e3a8517b92c58b06be1ce78d104c112c2f0f7a7dde37a75d1f934f0c4c"
+  license "MIT"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
```

Comment:
Do you need this? i.e. does `brew livecheck staq` return wrong results without this?